src/presentation/http/router/note.test.ts

```diff
@@ -49,4 +53,76 @@ describe('Note API', () => {
       expect(body).toStrictEqual({ message: 'Note not found' });
     });
   });
-});
+
+  describe('GET note/:notePublicId ', () => {
+    test('Returns note by public id with 200 status ' +
+    'when note is publicly available', async () => {
+      const expectedStatus = 200;
+      const correctID = 'Pq1T9vc23Q';
+
+      // TODO API should not return internal id and "publicId".
+      // It should return only "id" which is public id. Not implemented yet.
```

Comment:
please, use block comments and "todo" in jsdoc style: 

https://jsdoc.app/tags-todo.html